actionview/lib/action_view/helpers/tags.rb

```diff
@@ -30,6 +30,7 @@ module Tags # :nodoc:
         autoload :RangeField
         autoload :SearchField
         autoload :Select
+        autoload :SelectRenderer
```

Comment:
We can move this out of the `eager_autoload` block. It doesn't need to be eager loaded since it is included in classes that are eager loaded